docs/platform-engineers/addon/intro.md

```diff
@@ -100,175 +107,493 @@ metadata:
+				}
+				if parameter.clusters == _|_ {
+					clusterLabelSelector: {}
+				}
+			}
+		}]
+	}}
+```
 
-The result of the example will be as follows in YAML:
+If you execute command like below to enable the addon, resources will be deployed to `local` cluster and `cluster1` cluster.
+
+```shell
+$ vela addon enable <addon-name> clusters=local,cluser1
+```
+
+The render result will be:
 
 ```yaml
 kind: Application
-... 
-# application header in template
+metadata:
+  name: example
+  namespace: vela-system
 spec:
-  components:
-  - type: webservice
-    properties:
-    	image: "oamdev/vela-apiserver:v1.4.0"
-    traits:
-    - type: service-account
+  components: ...
+  policies:
+    - type: "topology"
+      name: "deploy-topology"
+      properties:
+        clusters:
+          - local
+          - cluster1
+```
+
+Since an empty `clusterLabelSelector` topology will choose all exist clusters as target clusters, you can just enable the addon without `clsuters` parameter to install it to all clusters:
+
+```shell
+$ vela addon enable <addon-name>
+```
+
+The render result is :
+
+```yaml
+kind: Application
+metadata:
+  name: example
+  namespace: vela-system
+spec:
+  components: ...
+  policies:
+    - type: "topology"
+      name: "deploy-topology"
       properties:
-        name: serviceAccountName
+        clusterLabelSelector: {}
 ```
 
-The detail of the example is [velaux](https://github.com/kubevela/catalog/tree/master/addons/velaux).
+#### Examples
 
-### Define parameter for addon
+Here are some existing addons made by community.
 
-When the resource is defined in CUE, you can also define parameters for addon by writing a `parameter.cue` file in the resources folder as shown below:
+* [OCM control plane](https://github.com/kubevela/catalog/blob/master/addons/ocm-hub-control-plane/template.yaml), this addon needn't change any content while enabling, so it chooses to use `template.yaml`.
+* [ingress-nginx](https://github.com/kubevela/catalog/tree/master/addons/ingress-nginx) addon is an example define a `template.cue` file, which can change some content of resources.
+
+### parameter.cue (Optional)
+
+In the example above, we used `parameter.namespace` parameter to set the name of namespace, and the `parameter.clusters` set the clusters that to be installed. We also need a parameter definition file (`parameter.cue`) to define what parameters the addon has. For examples:
 
 ```cue
-parameter: {
-  serviceAccountName: string
+parameter: {		
+  //+usage=clsuters install to
+  clsuters: [...string]
+  //+usage=namespace to create
+  namespace: string
 }
 ```
 
-It can cooperate with the resource file in CUE, here we have a CUE file `vela-apiserver.cue` :
+### `resources/` directory (Optional)
+
+Except writing every thing about an application in  one `temaplte.cue` or `template.yaml`, you also can split some resource description file to `resources` directory to avoid a very huge template file. File type in this dir can be YAML or CUE.
```

Comment:
every thing -> everything
in  one -> in one
some resource description files to the `resources` directly
You can place either YAML files or CUE files in this directory.